src/gridfs.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(dead_code, unused_variables)]
```

Comment:
This is unrelated, but removes the noise from `cargo check` and evergreen runs :)

Reply:
can you add a TODO RUST-1395 comment so we make sure to remove these?